hack/check

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export CGO_ENABLED=0
+
+find=find
+
+if [ "$(uname)" == "Darwin" ]; then
+	if ! command -v gfind ; then
+		echo "please install GNU Find"
+		echo ""
+		echo "    brew install findutils"
+		exit 1
+	fi
+	find=gfind
+fi
+
+pushd "$(dirname "$0")/.." >/dev/null
```

Comment:
Can you use this [style](https://github.com/pingcap/tidb-operator/blob/master/hack/run-e2e.sh#L7-L8) uniformly?